### PR TITLE
Revert "Leverage the TGS4 chat tagging system for the new game message (#41697)

### PR DESCRIPTION
**Pull request by MrsTonedOne**
This reverts commit db68cd1f9d51709af36d8ed2af71c18e01e32e53. (#41697)
@Cyberboss 

This seems to have broken tgs3 support, as no api function works, no chat command works, and scheduled dd restarts are not working causing the server to hang from memory leaks.

I say `this seems to have broken tgs3...` because this seems to be the only pr to touch the dmapi in the right time frame.

### CODE REVIEW

**Comment by MrStonedOne** [comment]
test merging this pr fixed the issue we were seeing, @Cyberboss I assume this is something that can be easily fixed instead of revertting.

**Comment by Jordie0608** [comment]
Merge conflict preventing updates.

**Comment by Cyberboss** [comment]
... Lummox may have :b:roken world params...
image

**Comment by Cyberboss** [comment]
It still the same thing while reverted, except it works for some reason...

File: code/modules/tgs/core/core.dm [hunk 1]
```diff
@@ -1,59 +1,66 @@
...
-	TGS_INFO_LOG("Activating API for version [version.deprefixed_parameter]")
-	var/datum/tgs_api/new_api = new api_datum(version)
-
-	TGS_WRITE_GLOBAL(tgs, new_api)
-
-	var/result = new_api.OnWorldNew(event_handler, minimum_required_security_level)
-	if(!result || result == TGS_UNIMPLEMENTED)
-		TGS_WRITE_GLOBAL(tgs, null)
-		TGS_ERROR_LOG("Failed to activate API!")
+		return /datum/tgs_api/latest
```
Comment (Cyberboss) [inline code/modules/tgs/core/core.dm]: Oof this is why, how long has this been broken?

Comment (Cyberboss) [reply code/modules/tgs/core/core.dm]: What's even weirder is how tf is this working since it activates the V4 API???

**Comment by Cyberboss** [comment]
https://secure.byond.com/forum/?post=2415144

**Comment by Cyberboss** [comment]
Cannot confirm this is the issue since I can't repro it outside of bash

**Comment by Cyberboss** [comment]
Nevermind, unrelated bug. Found the real issue

**Pull request closed without merging**